src/promptflow-devkit/promptflow/_sdk/entities/_run.py

```diff
@@ -155,7 +155,7 @@ def __init__(
         self.variant = variant
         self.run = run
         self._resume_from = kwargs.get("resume_from", None)
-        self._created_on = created_on or datetime.datetime.now()
+        self._created_on = created_on or datetime.datetime.now().astimezone()
```

Comment:
I remember there was some issue on the timezone in trace issue. Looks like this pr not target that？
cc @zhengfeiwang on this PR

Reply:
that one is an UX issue, and requires no client-side change